messaging/callbacks/msgDelivered.md

```diff
@@ -27,7 +27,7 @@ MMS and Group messages <b>don’t</b> currently support delivery receipts. Howev
 | message.direction     | `string` | Whether the message was sent from Bandwidth, or received by a Bandwidth number                                                                                                                                                                                                                                                                                      |
 | message.to            | `array`  | The phone number (or numbers) the message the message is sent to. On a POST, this can be a String, or an array of one or more numbers. In all other places, this will be an array.                                                                                                                                                                                  |
 | message.from          | `string` | The phone number the message was sent from                                                                                                                                                                                                                                                                                                                          |
-| message.text          | `string` | The text content of the message                                                                                                                                                                                                                                                                                                                                     |
+| message.text          | `string` | Empty text/string.                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Maybe add a small comment as to why we're having this be an empty string. Something like `Bandwidth no longer text contents for message delivered callbacks`?